src/data/accessibility-quiz.ts

```diff
@@ -186,8 +187,22 @@ const accessibilityQuiz = [
       "A computer program to scan an application for accessibility improvements.",
     Explanation:
       "A software application with assistive technology that attempts to convey what people with normal eyesight see on a display to their users via non-visual means, like text-to-speech. Developers can utilize a screen reader to understand how a webpage can be made accessible to such people and develop the page accordingly.",
-    Link: "https://www.freecodecamp.org/news/accessibility-testing-tools-for-ada-compliance/",
+    Link: "https://www.freecodecamp.org/news/accessibility-testing-tools-for-ada-compliance/"
   },
+  {
+    Question: "What is ARIA?",
+    Answer:
+      "ARIA defines the responsibilities, characteristics, and states for screenreaders in order to convey information about custom modules like carousels, accordions, and tabs.",
```

Comment:
Good but let's shorten it. shorter answers tend to work better in the quiz
```suggestion
      "ARIA defines characteristics for screen readers in order to convey information about custom modules like carousels, and tabs.",
```

Reply:
Kindly check if its proper now :)